build/selectorsyncset.yaml

```diff
@@ -141,7 +125,7 @@ objects:
       kind: ValidatingWebhookConfiguration
       metadata:
         annotations:
-          managed.openshift.io/inject-cabundle-from: openshift-validation-webhook/webhook-cert
+          service.beta.openshift.io/inject-cabundle: "true"
```

Comment:
Why are we updating the annotation prefix to `service.beta` ?

Reply:
@rogbas This is the annotation required by https://github.com/openshift/service-ca-operator

Reply:
In a literal sense the "why" is that this `build/selectorsyncset.yaml` file is generated by the `build/syncset.go` file (`make syncset`), and since the annotations in the `syncset.go` file changed, so too will the generated output.